modules/ivc_champva/spec/requests/ivc_champva/v1/forms/uploads_spec.rb

```diff
@@ -293,15 +293,74 @@
           end
         end
 
-        context 'when file uploads fail with other errors' do
+        context 'when file uploads fail with other errors retry once' do
```

Comment:
Please wrap this and the following test in a `context "Feature feature_flag=false"` block like in [this example](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample) so it's obvious that these tests can be removed when we take out the feature flag.

Reply:
Not needed  because you want to test with the feature in one error

Reply:
In all situations with flipper on or off.